City-of-Helsinki/drupal-hdbt
# UHF-X: Fix tab functionality on high school search

# Fix tab functionality on high school search
The high school search tabbing was broken on some instances and working on some. Now it should always work.

## What was done
* Changed how the views pre render function checks that we are on the correct paragraph.
* Changed the javascript so that the session storage is cleared on page reload.

## How to install
* Take latest KASKO instance with for example `make fresh`
* Require this: `composer require drupal/hdbt:dev-UHF-X_fix_high_school_search_tabbing`
* `make drush-cr`

## How to test
* Go to add a high school search and it should work correctly. You can also test the ones that are in testing if you use make fresh from testing.
* [This high school search should work now](https://nginx-kasvatus-koulutus-test.agw.arodevtest.hel.fi/fi/test-kasvatus-ja-koulutus/tutustu-ja-hae-lukioon)
